components/cf-typography/index.md

```diff
@@ -4,260 +4,232 @@ title:  "cf-typography"
             Consumer finance
         </a>
     </div>
-</div>
+</header>
 
+### Example in a cfgov-refresh Post Preview
 
-#### Alternate meta header arrangements
-<div class="block block__flush-top">
-    <div class="meta-header">
-        <a href="#" class="meta-header_right category-slug">
-            <span class="cf-icon cf-icon-credit-card"></span>
+<div class="o-post-preview">
+    <header class="m-meta-header">
+        <div class="m-meta-header_right">
+            <span class="a-date">
+                Nov 4, 2013
+            </span>
+        </div>
+        <div class="m-meta-header_left">
+            <span class="u-visually-hidden">Categories: </span>
+            <a href="#" class="a-heading a-heading__icon">
+                <span class="cf-icon cf-icon-credit-card"></span>
+                Consumer finance
+            </a>
+            |
+            <a href="#" class="a-heading a-heading__icon">
+                <span class="cf-icon cf-icon-credit-card"></span>
+                Consumer finance
+            </a>
+        </div>
+    </header>
+    <h3>
+        <a href="http://www.consumerfinance.gov/about-us/blog/administrador-del-acuerdo-de-provident-planea-ponerse-en-contacto-con-prestatarios-elegibles-proximamente/">
+            Administrador del Acuerdo de Provident planea ponerse en contacto con prestatarios elegibles próximamente
         </a>
-        <span class="meta-header_left date">
-            Nov 4, 2013
+    </h3>
+    <p>Prepaid cards can look and act like other types of cards, but they haven’t always had the same protections. Our new rule will help make prepaid cards and other prepaid accounts safer to use.</p>
+    <div class="post_meta">
+        <span class="o-post-preview_byline">
+            By <a href="http://www.consumerfinance.gov/about-us/blog/?filter1_authors=david-silberman">David Silberman</a>
         </span>
-    </div>
-</div>
 
-<div class="block block__flush-top">
-    <div class="meta-header">
-        <a href="#" class="meta-header_left category-slug">
-            <span class="cf-icon cf-icon-credit-card"></span>
-            Consumer finance
-        </a>
+        <div class="tags tags__hide-heading ">
+            <span class="u-visually-hidden">Topics:</span>
+            <ul class="tags_list">
+                <li class="tags_tag">
+                    <a href="http://www.consumerfinance.gov/about-us/blog/?filter1_topics=prepaid-cards" class="tags_link">
+                        <span class="tags_bullet" aria-hidden="true">•</span>
+                        Prepaid cards
+                    </a>
+                </li>
+                <li class="tags_tag">
+                    <a href="http://www.consumerfinance.gov/about-us/blog/?filter1_topics=rulemaking" class="tags_link">
+                        <span class="tags_bullet" aria-hidden="true">•</span>
+                        Rulemaking
+                    </a>
+                </li>
+            </ul>
+        </div>
```

Comment:
Do we care that this tag list doesn't resemble what it looks like on the site, which used code unique to cfgov-refresh?